core/services/workflows/engine.go

```diff
@@ -27,7 +27,11 @@ import (
 	"github.com/smartcontractkit/chainlink/v2/core/services/workflows/store"
 )
 
-const fifteenMinutesMs = 15 * 60 * 1000
+const (
+	fifteenMinutesMs             = 15 * 60 * 1000
```

Comment:
use time.Duration to avoid an ambiguity?

Reply:
The tradeoff is you cant use `const` as the time.Duration methods are runtime only